Source/WebKit/UIProcess/WebBackForwardCache.cpp

```diff
@@ -50,8 +50,8 @@ WebBackForwardCache::~WebBackForwardCache()
 
 inline void WebBackForwardCache::removeOldestEntry()
 {
-    ASSERT(!m_itemsWithCachedPage.isEmpty());
-    removeEntry(*m_itemsWithCachedPage.first());
+    ASSERT(!m_itemsWithCachedPage.isEmptyIgnoringNullReferences());
+    removeEntry(m_itemsWithCachedPage.first());
```

Comment:
Using tryTakeFirst() would make it even more robust.
if (auto* item = m_itemsWithCachedPage.tryTakeFirst())
    item->setBackForwardCacheEntry(nullptr);